### PR TITLE
Hiral_Assignment_20

### PR DESCRIPTION
**Pull request by purohit31**
### Assignment Number :1234:

Hiral_Assignment_20

### Assignment Description :notebook:

print the first 5 even numbers using for loop, while loop, do while loop

### Evidence :file_folder:

image

### Checklist before requesting a review :ballot_box_with_check:

- [x] I have performed a self-review of my code
- [x] My changes generate no compile time errors

### CODE REVIEW

**Comment by purohit31** [comment]
plz check commit "Hiral_Assignment_20_2". which i have updated.

File: src/hiral_purohit/WhileDowhileLoop.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		while(i <= num2) {
+			if(i % 2 == 0) {
+				System.out.println(i);
+				count++;
+				if(count >= 5) {
+					break;
+				}
+			}
+			i++;
+			}
```
Comment (MargiVegada) [inline src/hiral_purohit/WhileDowhileLoop.java:30]: Brackets are not aligned

**Review by MargiVegada — APPROVED** [review]
Need to work on indentation

File: src/hiral_purohit/WhileDowhileLoop.java (hunk 1 above)
Comment (purohit31) [reply src/hiral_purohit/WhileDowhileLoop.java]: i will take care of it next time

**Pull request merged**